### ISSUE TITLE
Feature Request: Separate the delay before the start of a first war in a chain from the time between wars in the chain

### ISSUE THREAD

**Issue by lisushka** [lisushka/Winnie_Bot#26]
**Describe the solution you'd like**
It would be nice to be able to set a delay on the start of a chain war separately to the time between each war, which is what I believe the default initial delay is. It may mess up your current parameter parsing system but I think it would be useful. E.g. I want to wait 10 minutes, than start 3 15-minute chain wars with a 5-minute delay between each.

**Comment by Rincewing** [comment]
Bubbleduck here, I gave it some thought and the only solution I've come up with with right now is to add another layer of parsing for the time variable. It could be generalised as well to allow for specifying multiple different durations between wars.

So the original structure of the command is the same, !chainwar <number of wars> <duration> [time between wars [name]],
but [time between wars] can be expanded out to [time after first war[/time after second war[/time after third war[/...]]]], taking the last value entered as the time taken between all remaining wars.

This does add an extra layer of complexity to the command but the benefit is that users can keep going with no changes to the existing structure and learn the new method when they need.

Usage examples:
!chainwar 4 10 10/5 would set a duration of 10 minutes initial wait, then start four wars of 10 minutes with 5 minutes between each.
!chainwar 3 15 1/10/5 would set a 1 minute initial wait, then start a 15 minute war, wait another 10 minutes, start a second 15 minute war and finally wait 5 minutes before the last war. If the /5 was excluded the final wait would have been 10 minutes.

**Comment by lisushka** [comment]
Thank you.  I'm going to look into the feasibility of this.

**Comment by wcmckee** [comment]
Hi. I wrote a python function to do this. It just creates single wars/sprints but allows the user to specify the amount of sprints, name of each sprint and the time inbetween. 

```python
def createseriessprint(namesprints, length, timestart, waitbetween, amountsprint):
    #nameofsprints - list of the names of 
    spnam = namesprints.split(',')
    timetosleep = (waitbetween * 2) + length
    sleepsecs = timetosleep * 60 
    
    #return(sleepsecs)
    for cr in range(amountsprint):
        print(str(cr) + spnam[cr])
        createsprint('war', length, timestart, spnam[cr])
        Hook(hook_url=webhook, username="Pinny",
         avatar_url='https://pbs.twimg.com/profile_images/649004216690675712/UAcrP6ol_400x400.png',
         content="war {} is over. The next starting in {}".format(spnam[cr], timestart)).execute()
        time.sleep(sleepsecs)
```

**Comment by lisushka** [comment]
Resolved with pull request #64.  Instructions for using the granular command are in the pull request notes.
